packages/app-extrinsics/src/Extrinsic.tsx

```diff
@@ -82,7 +84,9 @@ class Extrinsic extends React.PureComponent<Props, State> {
           isValid &&
           !isUndefined(values[index]) &&
           !isUndefined(values[index].value) &&
+          (values[index].type === 'Balance' ? checkValueBitLength(values[index].value as BN) : true) &&
```

Comment:
Nested condition, would make sense to make the code clean and split this condition out into `isValidLength` and then just `&& isValidLength`